misago/users/management/commands/anonymizeoldips.py

```diff
@@ -5,7 +5,6 @@
 from misago.core.utils import ANONYMOUS_IP
 from misago.users.signals import anonymize_old_ips
 
-
```

Comment:
This change is wrong. Please keep two blank lines between last import in your python file, and anything else.